model-optimizer/extensions/front/onnx/AttributedSliceToSlice_test.py

```diff
@@ -0,0 +1,63 @@
+"""
+ Copyright (C) 2018-2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import unittest
+
+import numpy as np
+from generator import generator, generate
+
+from extensions.front.onnx.AttributedSliceToSlice import AttributedSliceToSliceReplacer
+from mo.utils.ir_engine.compare_graphs import compare_graphs
+from mo.utils.unittest.graph import build_graph, regular_op_with_empty_data, result, const, connect_on_front
+
+
+@generator
+class SliceReplacerTest(unittest.TestCase):
+    @generate(*[
+       {'op': 'AttributedSlice', 'type': None, 'starts': np.array([0, 0]), 'ends': np.array([1, -1])},
```

Comment:
Please, add more non-trivial tests (when values are not 0s and 1s)

Reply:
This transformation depends only from the value of `axis`. Either it's set either not. The actual values does not affect transformation. More unit-tests will be redundant.